src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,87 @@
+package duke.task;
+
+public class Task {
+    /**
+     * Represent a general Task.
+     */
+    boolean status;
+    String des;
+
+    /**
+     * Constructor to create a generic Task.
+     *
+     * @param status
+     * @param des
+     */
+    public Task(boolean status, String des) {
+        this.status = status;
+        this.des = des;
+    }
+
+    /**
+     * Empty constructor to create a generic Task
+     */
+    public Task() {
+
+    }
+
+    /**
+     * Method to configure information read from file, into Task objects for TaskList
+     *
+     * @param des
+     */
+    public void configure(String[] s) {
+        setDes(s[0]);
+    }
+
+    /**
+     * Method to return description of Task.
+     *
+     * @return String representation of description of Task
+     */
+    public String getDes() {
+        return this.des;
+    }
+
+    /**
+     * Method to define the description of the Task
+     *
+     * @param des Description of task
+     */
+    public void setDes(String des) {
+        this.des = des;
+    }
+
+    /**
+     * Method to return status of Task
+     *
+     * @return Boolean to represent status of task. True for done, false for not done.
+     */
+    public boolean getStatus() {
+        return status;
+    }
```

Comment:
Consider using the method isStatus() instead of getStatus() to follow the coding convention